rescript/plugin_setup.py

```diff
@@ -1235,6 +1238,149 @@
     ]
 )
 
+
+datatypes_metadata = [
+    "antibiotics",
+    "enzyme_class_ref",
+    "epitope",
+    "epitope_assay",
+    "experiment",
+    "bioset",
+    "bioset_result",
+    "gene_ontology_ref",
+    "genome",
+    "strain",
+    "genome_amr",
+    "feature_sequence",
+    "genome_feature",
+    "genome_sequence",
+    "id_ref",
+    "misc_niaid_sgc",
+    "pathway",
+    "pathway_ref",
+    "ppi",
+    "protein_family_ref",
+    "sequence_feature",
+    "sequence_feature_vt",
+    "sp_gene",
+    "sp_gene_ref",
+    "spike_lineage",
+    "spike_variant",
+    "structured_assertion",
+    "subsystem",
+    "subsystem_ref",
+    "taxonomy",
+    "protein_structure",
+    "protein_feature",
+    "surveillance",
+    "serology"
+]
+
+plugin.methods.register_function(
+    function=fetch_genomes_bv_brc,
+    inputs={},
+    parameters={
+        'rql_query': Str,
+        'genome_ids': List[Str],
+    },
+    outputs=[('genomes', GenomeData[DNASequence])],
+    input_descriptions={},
+    parameter_descriptions={
+        'rql_query': 'Query in RQL format. Check '
+                     'https://www.bv-brc.org/api/doc/genome_sequence '
+                     'for documentation.',
+        'genome_ids': 'List of genome IDs from BV-BRC.',
+
+    },
+    output_descriptions={
+        'genomes': 'genomes',
+    },
+    name='fetch genomes',
+    description="fetch genomes",
+)
+
+
+plugin.methods.register_function(
+    function=fetch_metadata_bv_brc,
+    inputs={},
+    parameters={
+        'data_type': Str % Choices(datatypes_metadata),
+        'rql_query': Str
+    },
+    outputs=[('metadata', ImmutableMetadata)],
+    input_descriptions={},
+    parameter_descriptions={
+        'data_type': 'BV-BCR data type. Check https://www.bv-brc.org/api/doc/ '
+                     'for documentation.',
+        'rql_query': 'Query in RQL format. Check '
+                     'https://www.bv-brc.org/api/doc/ for documentation.'
+    },
+    output_descriptions={
+        'metadata': 'metadata'},
+    name='Fetch BV-BCR metadata.',
+    description="Fetch BV-BCR metadata for a specific data type with an RQL "
```

Comment:
Again, please provide a detailed description of what this action retrieves and what it outputs. What is the downstream use case for this? Can you provide an example?